local-cli/runAndroid/runAndroid.js

```diff
@@ -263,17 +263,18 @@ function runOnAllDevices(args, cmd, packageNameWithSuffix, packageName, adbPath)
 }
 
 function startServerInNewWindow(port) {
-  const scriptFile = /^win/.test(process.platform) ?
+  const isWin = /^win/.test(process.platform);
+  const scriptFile = isWin ?
     'launchPackager.bat' :
     'launchPackager.command';
   const scriptsDir = path.resolve(__dirname, '..', '..', 'scripts');
   const launchPackagerScript = path.resolve(scriptsDir, scriptFile);
   const procConfig = {cwd: scriptsDir};
   const terminal = process.env.REACT_TERMINAL;
 
-  // setup the .packager.env file to ensure the packager starts on the right port
-  const packagerEnvFile = path.join(__dirname, '..', '..', 'scripts', '.packager.env');
-  const content = `export RCT_METRO_PORT=${port}`;
+  // setup the .packager.(env|bat) file to ensure the packager starts on the right port
```

Comment:
Can we decouple file names and command names into separate variables/constants?

Reply:
I'm not sure what you want here? are you looking for new `const` variables, one for `.packager.bat` and one for `.packager.env`?

I'd have preferred to use just a single filename, unfortunately, Windows doesn't have a `source .env` equivalent, you can get the same result by using a batch file, but the batch files must have a .bat extension to be invoked with call from the parent script...